control-plane/agents/src/bin/ha/cluster/switchover.rs

```diff
@@ -173,6 +176,19 @@ impl SwitchOverRequest {
         Ok(())
     }
 
+    /// Deletes Swtichover request from Etcd.
+    async fn delete_switchover(&mut self, etcd: &EtcdStore) -> Result<(), anyhow::Error> {
+        self.start_op(Stage::Successful, etcd).await?;
+        info!(volume.uuid=%self.volume_id, "Deleting Switchover request from Etcd as request completed successfully");
```

Comment:
Will use lowercase in all comments/docs

Reply:
No, I want to delete them. And the existing code on startup just ignores completed entries.

Reply:
We delete the swicthover request when stage is updates as Successful. ideally there should be no requests in Successful stage at startup. It can happen if stage is updated for a request but before deleting from etcd cluster agent crashed. 

In this case also. At startup if we encounter any Successful request. We would delete that request. Sortof like a cleanup workflow. Is that okay?